internal/workermanager/discovery_worker.go

```diff
@@ -108,6 +109,10 @@ func (m *Manager) SyncAssets(ctx context.Context, job worker.JobSpec) error {
 
 		for _, ast := range assets {
 			if err := m.discoveryRepo.Upsert(ctx, ast); err != nil {
+				if strings.Contains(err.Error(), "illegal_argument_exception") {
+					m.logger.Error(err.Error())
```

Comment:
let's log the asset information here like urn for make easier to identify the issue

Reply:
log will looks like this: `discovery error: IndexDoc: doc ID 'c125cca2-0df8-43d6-b6c8-9111f6f7b202': index 'bigquery': elasticsearch code 'illegal_argument_exception': mapper [data.preview_rows] of different type, current_type [long], merged_type [text]` I think doc ID is sufficient to identify. Wdyt?

Reply:
i see, it's already handled in the repo, then it's look fine